packages/apt/test.sh

```diff
@@ -150,22 +150,18 @@ fi
 
 echo "::group::Upgrade"
 
-sudo apt purge -V -y ${pgroonga_package}
-
-# Disable upgrade test for first time packages.
-# TODO: Automate this. We don't want to maintain this manually.
-# case ${postgresql_version} in
-#   16) # TODO: Remove this after 3.1.4 release.
-#     exit
-#     ;;
-# esac
-
-sudo apt install -V -y ${pgroonga_package}
-createdb upgrade
-psql upgrade -c 'CREATE EXTENSION pgroonga'
-apt install -V -y \
-  ${repositories_dir}/${os}/pool/${code_name}/*/*/*/*_${architecture}.deb
-psql upgrade -c 'ALTER EXTENSION pgroonga UPDATE'
+sudo apt-get purge -V -y ${pgroonga_package}
```

Comment:
他のところで`apt`を使っているので`apt`でいいっす。

```suggestion
sudo apt purge -V -y ${pgroonga_package}
```

Reply:
fix: [6a32183](https://github.com/pgroonga/pgroonga/pull/484/commits/6a321834d69f13358730cdcfc0c41246f9e12784)
他の部分と合わせて、`apt` を使う方針に修正しました。